code/src/cljs/sixsq/nuvla/ui/deployments/utils.cljs

```diff
@@ -101,6 +103,7 @@
   (case state
```

Comment:
This function is not working as expected for STARTING and PENDING values. Case only accept constant values. https://clojuredocs.org/clojure.core/case